src/libs/antares/study/parameters.cpp

```diff
@@ -1141,6 +1156,11 @@ void Parameters::fixBadValues()
         nbTimeSeriesSolar = 1;
 }
 
+Yuni::uint64 Parameters::memoryUsage() const
```

Comment:
only used in src/libs/antares/study/study.cpp:216

Should be removed if it only returns 0

Reply:
We could return something like `sizof(Parameters)`, add the size of pointed members, vectors, arrays, etc.

Reply:
done !